products/compute/ansible_facts.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Ansible::Facts::Config
+manifest: !ruby/object:Provider::Ansible::Manifest
+  metadata_version: '1.1'
+  status:
+    - preview
+  supported_by: 'community'
+  requirements:
+    - python >= 2.6
+    - requests >= 2.18.4
+    - google-auth >= 1.3.0
+  version_added: '2.7'
+  author: Google Inc. (@googlecloudplatform)
+overrides: !ruby/object:Provider::ResourceOverrides
+  Instance: !ruby/object:Provider::Ansible::ResourceOverride
+    generate_facts: true
```

Comment:
Hmm, every resource that's listed in this file is going to have generate_facts: true, right? Is there a way we can avoid having to explicitly say that?

Reply:
I just wanted to do that temporarily. My intention is to set that to true by default (and probably merge this provider into the regular Ansible provider).

Reply:
This seems very much like `enabled: true`, right?